src/components/common/stars.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+  import range from '@/utils/range.js';
+
+  export let value;
+  export let max = 5;
+</script>
+
+<div class="stars">
+  {#each [...range(max)] as i}
+    {#if i < value}
```

Comment:
Remember that SVGs are actually substituted, I wouldn't play with it much. It might work, but it's best to do this if we will ever move to component-based icons

Reply:
And what about the [`StarRating` component](https://github.com/illright/attractions/blob/develop/attractions/star-rating/star-rating.svelte)?

Reply:
`StarRating` is interactive, it's meant to select a rating rather than display it. This one only displays

Reply:
maybe it is worth adding a `readOnly` prop that would just display it?

Reply:
Nah, those are completely different components with different markup. Perhaps what we could do is move the read-only stars to Attractions